docs/getting-started/transactions/curl/datatx.md

```diff
@@ -207,9 +210,9 @@ use the `base64_decoder` utility.
 The result contains the base64 decoded `value` associated with the key and also the `access_control` and `version` as part of the
 `metadata`. In order to verify the signature, do not use the output of `base64_decoder` utility but the `cURL` output.
 
-## (3) Updation of an existing state
+## 2) Updation of an existing state
```

Comment:
Updation is only valid India... "Updating an existing state"

Reply:
I will do this at once shot https://github.com/hyperledger-labs/orion-server/issues/284 It has been used in other places too